config/sync/block.block.subtheme_videogame_page_title.yml

```diff
@@ -17,4 +19,9 @@ settings:
   label: 'Page title'
   provider: core
   label_display: '0'
-visibility: {  }
+visibility:
+  request_path:
+    id: request_path
+    pages: '/user/*'
```

Comment:
Can you check if this is the expected visibility of page-title block? Should it only be visible on the user pages?